### DISCUSSION TITLE
Welcome to 101Cats Discussions!

### DISCUSSION THREAD

**Discussion by martinbradford** [martinbradford/101Cats#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vk4tux** [comment]
Thanks Martin, I don't think these get emailed ?

73

vk4tux

**Reply by martinbradford** [reply]
Your message was emailed to me...

**Reply by mdblack98** [reply]
You have to go to the Code page and use the "Watch" pulldown to choose what to subscribe to.  You can Customize for example if you want only discussions or choose "All Activity" to see code change and such too.

**Comment by martinbradford** [comment]
This is good!

I've never used GitHub for anything more serious than basic source control before, so I really don't understand these advanced features - will need to experiment.

Are you also able to update the Wiki?

Martin

**Comment by JoeDaring** [comment]
Hellò Martin...
Looks like GitHub Discussions may do the job!

**Reply by martinbradford** [reply]
can you check if you can edit the wiki? that would be very helpful

**Reply by JoeDaring** [reply]
I Martin... gone to check the Wiki, but I see no way to edit the stuff you have created...

**Reply by martinbradford** [reply]
I think you have to be registered as a collaborator - I will add you to the list...

**Comment by I2RNJ** [comment]
Hi Martin, nice application, really.
I am testing version 1.0.9.0 without any issue, but maybe something is missing, unless it could be my misunderstanding.
Under "Antenna 3 Function" I cannot see TRX, while on FTDX101 menu, the function is selectable.
73
Roberto
I2RNJ

**Reply by martinbradford** [reply]
Hi Roberto and thanks...

I will check this later. The "Antenna 3 Function" box is a recent addition and lightly tested at the moment. I use the Antenna 3 socket to connect an external SDR - if it is accidentally set to TRX, then the first time you transmit, you will destroy the SDR! I intended to add a configuration item to block this, but I had a serious computer failure that took me a couple of weeks to recover from and I completely forgot to add that functionality... I will work on it later today, or tomorrow. Will announce it here when it's done.

Martin

**Reply by martinbradford** [reply]
P.S. can you try adding an Issue for this - just to make sure that we have that configured correctly?

**Comment by vk4tux** [comment]


**Reply by martinbradford** [reply]
Thanks Adrian. You are looking in the source repository - I'm still working on changes and will only bump the version number when I create a new installer that will be distributed through this repo. While you are on-line, can you check if you can update the wiki in this repository? I'm keen that other collaborators should be able to contribute towards the documentation!

**Reply by martinbradford** [reply]
And I see that you can! Excellent! Feel free to make any additions that you think appropriate...

**Comment by martinbradford** [comment]
A preliminary web site, www.101cats.co.uk, is also live now. It is hosted through GitHub and he content is automatically generated from the contents of the /doc directory of this repository. I'm still learning how to use GitHub Pages - functionality is good, but documentation could be better!
